packages/cli/src/cog/__test__/builder.test.ts

```diff
@@ -39,7 +39,7 @@ o.spec('Builder', () => {
 
         o('bounds', async () => {
             const localTiff = new CogSourceFile('/local/file.tiff');
-            const s3Tiff = new CogSourceAwsS3('bucket', 's3://file.tiff');
+            const s3Tiff = new CogSourceAwsS3('bucket', 's3://file.tiff', null as any);
```

Comment:
Maybe a comment as to why this is needed?

Reply:
This was a API change to force you to push in a S3 object into the sources, this test overrides the reading of the tiffs.

if this was created with `CogSourceAwsS3.create` you dont need the `as any`